node/cli/src/command.rs

```diff
@@ -52,6 +50,7 @@ impl PartialEq for ChainIdentity {
 			(ChainIdentity::Frequency, ChainIdentity::Frequency) => true,
 			(ChainIdentity::FrequencyRococo, ChainIdentity::FrequencyRococo) => true,
 			(ChainIdentity::FrequencyLocal, ChainIdentity::FrequencyLocal) => true,
+			(ChainIdentity::FrequencyDev, ChainIdentity::FrequencyDev) => true,
```

Comment:
It is not from this PR, but now that I look at this, I think, why is this not derived off of `enum ChainIdentity`?

Reply:
It's not clear why this was added (several months ago) or why it's needed.